src/deepsparse/pipeline.py

```diff
@@ -220,8 +227,39 @@ def __call__(self, *args, **kwargs) -> BaseModel:
                 f"Outputs of {self.__class__} must be instances of "
                 f"{self.output_schema} found output of type {type(pipeline_outputs)}"
             )
+        timer.stop(InferencePhases.POST_PROCESS)
+        timer.stop(InferencePhases.TOTAL_INFERENCE)
+
+        if not monitoring:
+            return pipeline_outputs
+
+        else:
+            inference_timing = InferenceTimingSchema(**timer.build())
+            return pipeline_outputs, pipeline_inputs, engine_inputs, inference_timing
```

Comment:
Do we know how exactly will pipeline_inputs/engine_inputs be used? I know monitoring data distribution was discussed previously, but what does that actually look like?

Reply:
This remains to be defined.